### PR TITLE
Revert "(maint) Update changelog for 2.1.6 release"

### PR DESCRIPTION
**Pull request by mwaggett**
This reverts commit fe5919e5a52ba3b7e6f20b5e7a6fbfaa2006f46e.
This version was never released because we ended up reverting the only change
it would have included.

Please add all notable changes to the "Unreleased" section of the CHANGELOG.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -761,4 +755,4 @@ This release contains bug fixes and AIO path changes.
 [2.1.5]: https://github.com/puppetlabs/ezbake/compare/2.1.4...2.1.5
 [2.1.4]: https://github.com/puppetlabs/ezbake/compare/2.1.3...2.1.4
 [2.1.3]: https://github.com/puppetlabs/ezbake/compare/2.1.2...2.1.3
-[2.1.2]: https://github.com/puppetlabs/ezbake/compare/2.1.1...2.1.2
\ No newline at end of file
+[2.1.2]: https://github.com/puppetlabs/ezbake/compare/2.1.1...2.1.2
```
Comment (sarameisburger) [inline CHANGELOG.md]: Should this be removing 2.1.6?

Comment (mwaggett) [reply CHANGELOG.md]: I think this is one of those weird newline changes, but yeah, the 2.1.6 line should also be removed, good catch. not sure why the revert didn't get that..

Comment (mwaggett) [reply CHANGELOG.md]: ahh, because we used to not have links, so that's a separate commit. updating now.

File: CHANGELOG.md [hunk 2]
```diff
@@ -756,9 +750,8 @@ This release contains bug fixes and AIO path changes.
 ## 0.1.0 - 2015-01-13
  * Rewrite ezbake to follow leiningen plugin application model.
 
-[Unreleased]: https://github.com/puppetlabs/ezbake/compare/2.1.6...HEAD
-[2.1.6]: https://github.com/puppetlabs/ezbake/compare/2.1.5...2.1.6
+[Unreleased]: https://github.com/puppetlabs/ezbake/compare/2.1.5...HEAD
```
Comment (mwaggett) [inline CHANGELOG.md]: @sarameisburger updated!

**Pull request merged**